src/classes/TaskItemController.cls

```diff
@@ -0,0 +1,31 @@
+public with sharing class TaskItemController {
+
+	@AuraEnabled
+	public static List<Task__c> getTaskItems()
+	{
+		return [SELECT Id, Name, Status__c, Description__c, Contact__c, Contact__r.Name, Contact__r.Total_Points_Awared__c, RecordTypeId, RecordType.Name FROM Task__c];
+	}
+
+	@AuraEnabled
+	public static Contact getTaskContactApex(Task__c task)
+	{
+		//ID contactId = task.Contact__c;
+		Contact taskContact = [SELECT Id, Name, Total_Points_Awared__c FROM Contact where Id =: task.Contact__c ];
+		return taskContact;
+	}
+
+	@AuraEnabled
+	public static void updateTask(Task__c updateTask){
+
+		update updateTask;
+	}
+
+	@AuraEnabled
+	public static void ContactHabitDone(Contact contactTask){
```

Comment:
ContactHabitDone does not let me understand what does this method do. same for "contactTask". Please use mnemonic names that actually have a meaning. For example if you call this method simply "updateContact", it makes the intention quite clear. Likewise, calling the method's parameter simply "contact" and adding some comments on the method as to when are you calling it should be good enough.

Reply:
Ok, I've changed both.